extensions/wikia/WikiaNewFiles/WikiaNewFilesGallery.class.php

```diff
@@ -49,7 +49,7 @@ public function addImages( array $images ) {
 			}
 
 			$caption = wfMessage( 'wikianewfiles-uploadby' )->rawParams( $ul )->params( $ut )->escaped();
-			$caption .= "<br />\n<i>$timeago</i><br />\n";
+			$caption .= " • <i>$timeago</i><br />\n";
```

Comment:
Don't we want to allow translators to make decision about the order of these labels?